src/coreclr/ToolBox/superpmi/superpmi-shared/methodcontext.cpp

```diff
@@ -4167,7 +4167,9 @@ LONG* MethodContext::repGetAddrOfCaptureThreadGlobal(void** ppIndirection)
     if ((GetAddrOfCaptureThreadGlobal == nullptr) || (GetAddrOfCaptureThreadGlobal->GetIndex((DWORD)0) == -1))
     {
 #ifdef sparseMC
-        LogDebug("Sparse - repGetAddrOfCaptureThreadGlobal returning 0xCAFE0001");
+        LogDebug("Sparse - repGetAddrOfCaptureThreadGlobal returning nullptr and 0xCAFE0001");
+        if (ppIndirection != nullptr)
+            *ppIndirection = nullptr;
```

Comment:
why not set it to another hackish value, maybe the same `0xCAFE0001`?

Reply:
I could've done this as well. 

But it was not clear to me whether such approach is better or worse than simply setting to `null`.